.github/workflows/prepare_release.yml

```diff
@@ -156,11 +156,15 @@ jobs:
           version=$(echo $file_name | sed -e "s/^pulumi-${{ matrix.provider }}-kotlin-//" -e "s/-javadoc.jar$//")
           mkdir -p ./docs/${{ matrix.provider }}/$version
           mv ./build/dokka/pulumi${{ steps.provider.outputs.capitalized }}${{ matrix.major-version }}Javadoc/* ./docs/${{ matrix.provider }}/$version
+      - name: Install GitHub CLI # TODO: move this step to runner VM setup
+        if: ${{ steps.check-for-release.outputs.is_release == 'true' && !contains(matrix.runs-on, 'ubuntu-latest') }}
+        uses: dev-hanz-ops/install-gh-cli-action@v0.1.0
+      - name: Install missing tools # TODO: move this step to runner VM setup
+        if: ${{ steps.check-for-release.outputs.is_release == 'true' && !contains(matrix.runs-on, 'ubuntu-latest') }}
+        run: yum -y install wget zip
```

Comment:
Default GH runners have gh, wget, and zip installed by default, but our runners don't. We can move this step to the VM setup script in the future